apps/src/applab/extraControlRows.html.ejs

```diff
@@ -1,31 +1,36 @@
 <% var msg = require('../../locale/current/common') %>
 <% var applabMsg = require('../../locale/current/applab') %>
 
-<% if (debugButtons) { %>
-<div>
-  <div id="debug-buttons" style="display:inline;">
-    <button id="pauseButton" class="share">
-      <%= applabMsg.pause() %>
-    </button>
-    <button id="stepInButton" class="share">
-      <%= applabMsg.stepIn() %>
-    </button>
-    <button id="stepOverButton" class="share">
-      <%= applabMsg.stepOver() %>
-    </button>
-    <button id="stepOutButton" class="share">
-      <%= applabMsg.stepOut() %>
-    </button>
-    <button id="viewDataButton" class="share" style="display:none;">
-      <%= applabMsg.viewData() %>
-    </button>
-    <button id="designModeButton" class="share">
-      <%= applabMsg.designMode() %>
-    </button>
+<div id="debug-area">
```

Comment:
Please keep this div hidden in Design Mode by adding the following code to Applab.toggleDesignMode:

  var debugArea =  document.getElementById('debug-area');
  debugArea.style.display = enable ? 'none' : 'block';